notebooks/samples/105 - Regression with DataConversion.ipynb

```diff
@@ -0,0 +1,211 @@
+   "source": [
+    "Split the datasest into train and test sets."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "train, test = flightDelay.randomSplit([0.75, 0.25])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Train a regressor on the dataset with l-bfgs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from mmlspark import TrainRegressor, TrainedRegressorModel\n",
+    "from pyspark.ml.regression import LinearRegression\n",
+    "from pyspark.ml.feature import StringIndexer\n",
+    "\n",
+    "# Convert columns to categorical\n",
+    "catCols = [\"Carrier\", \"DepTimeBlk\", \"ArrTimeBlk\"]\n",
+    "for catCol in catCols:\n",
+    "    siModel = StringIndexer(inputCol=catCol, outputCol=catCol+\"Tmp\").fit(train)\n",
+    "    trainCat = siModel.transform(train).drop(catCol).withColumnRenamed(catCol+\"Tmp\", catCol)\n",
+    "    testCat = siModel.transform(test).drop(catCol).withColumnRenamed(catCol+\"Tmp\", catCol)\n",
+    "lf = LinearRegression().setSolver(\"l-bfgs\").setRegParam(0.1).setElasticNetParam(0.3)\n",
+    "model = TrainRegressor(model=lr, labelCol=\"ArrDelay\").fit(trainCat)\n",
+    "model.write().overwrite().save(\"flightDelayModel.mml\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Score the regressor on the test data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "flightDelayModel - TrainedRegressorModel.load(\"floightDelayModel.mml\")\n",
+    "scoredData = flightDelayModel.transform(testCat)\n",
+    "scoredData.limit(10).toPandas()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Compute model metrics against the entire scored dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from mmlspark import ComputeModelStatistics\n",
+    "metrics = ComputeModelStatistics().transform(scoredData)\n",
+    "metrics.toPandas()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, compute and show per-instance statistics, demonstrating the usage of `ComputePerInstanceStatistics`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from mmlspark import ComputePerInstanceStatistics\n",
+    "evalPerInstance = ComputePerInstanceStatistics().transform(scoredData)\n",
+    "evalPerInstance.select(\"ArrDelay\", \"Scores\", \"L1_loss\", \"L2_loss\").limit(10).toPandas()"
+   ]
+  }
+ ],
+ "metadata": {
```

Comment:
I believe you can remove these.. please check with @elibarzilay

Reply:
? This seems to be consistent with the other sample notebooks?